Omax.cfg

```diff
@@ -0,0 +1,10 @@
+-Ofast \
```

Comment:
as `-Ofast` has now been depreciated and this now will emit a warning (https://github.com/llvm/llvm-project/pull/98736/files), I think it would make sense to change this to `-O3 -ffast-math`.

Reply:
Done!